src/components/site/Breadcrumbs.md

```diff
@@ -2,16 +2,17 @@
 
 ```jsx
 import Section from "../basic/containers/Section";
-import BreadcrumbItem from "./Breadcrumbitem";
 import BreadcrumbLinkItem from "./BreadcrumbLinkitem";
+import BreadcrumbLastItem from "./BreadcrumbLastItem";
+
 <Section>
   <Breadcrumbs>
-    <BreadcrumbLinkItem text="Departments" link="/departments" />
+    <BreadcrumbLinkItem text="Departments" link="/departments/health" />
     <BreadcrumbLinkItem
       text="Department of Health"
       link="/departments/health"
     />
-    <BreadcrumbItem text="Adoptable Pets" />
+    <BreadcrumbLastItem text="Adoptable Pets" link="/departments/health" />
```

Comment:
Would prefer to keep this `BreadcrumbLinkItem` if possible.

Reply:
As mentioned in my last comment, updated to the new version of `BreadcrumbLinkItem`.